src/utils.ts

```diff
@@ -142,3 +142,14 @@ export function convertNewlineToBr(text: string) {
 export function createMarkup(text: string) {
   return { __html: text };
 }
+
+export const getLocalStorage = (key: string) => {
+  const value = localStorage.getItem(key);
+  if (value !== null) {
+    return JSON.parse(value);
+  }
+};
```

Comment:
요거 리턴타입 `any`로 나오지 않나유??

```suggestion
export function getLocalStorage<T>(key: string) {
  const value = localStorage.getItem(key);
  if (value !== null) {
    return JSON.parse(value) as T;
  } else {
    return null;
  }
};
```
처럼 가면 `T | null`을 리턴하도록 만들 수 있을 것 같아유!

Reply:
오 좋씁니다!!!